R/simulations_contrast.R

```diff
@@ -0,0 +1,76 @@
+#' Create simplest simulation, cj = mu 1
+#' data used for contrast analysis
+#' @param nsamp number of samples of each type
+#' @param ncond number of conditions
+#' @param err_sd the standard deviation of the errors
+#' @details There is no true deviation exists in this case
+#' @export
+
+sim.contrast1 = function(nsamp = 100, ncond = 5, err_sd=sqrt(0.5)){
```

Comment:
can you rename these sim_contrast1 and sim_contrast2 for consistency with other function names (ie use _ instead of . in function names)